crates/pint-deploy/src/main.rs

```diff
@@ -94,16 +97,24 @@ async fn run(args: Args) -> anyhow::Result<()> {
             let contract_path = profile_dir.join(&pinned.name).with_extension("json");
             let (contract, programs) = contract_from_path(&contract_path).await?;
             print_deploying(&pinned.name, &contract);
+
             let output = builder_client
                 .deploy_contract(&big_bang, &contract, &programs)
                 .await?;
             print_received(&output);
+            submit(&contract, &builder_address).await?;
         }
     }
 
     Ok(())
 }
 
+async fn submit(contract: &Contract, builder_address: &str) -> Result<(), anyhow::Error> {
```

Comment:
The last part can be left to a follow-up PR if you wish, I'm happy either way

Reply:
I will leave the last part for a follow-up to avoid scope-creep in this PR.
I will open an issue for it though, assign to myself and add it to my shortlist of tasks. :)

Reply:
https://github.com/essential-contributions/essential-integration/issues/146